frontend/src/hooks/useNavigation.ts

```diff
@@ -33,11 +33,11 @@ export function useNavigation(): [string, INavigation[]] {
   }, [navigation, location, menu])
 
   const menuItems: INavigation[] = [
-    { label: 'This Device', path: '/devices', match: '/devices', icon: 'hdd', show: remoteUI },
+    { label: 'This Device', path: '/devices', match: '/devices/:any?/:any?/:any?', icon: 'hdd', show: remoteUI },
```

Comment:
why these params are `any` ?

Reply:
Because it will match :any as an optional param to create the active state of the menu item -it doesn't matter what the key is.